src/Symfony/Bundle/TwigBundle/Resources/views/Exception/traces.html.twig

```diff
@@ -22,7 +22,7 @@
         <div id="trace-html-{{ index }}" class="sf-toggle-content">
         {% set _is_first_user_code = true %}
         {% for i, trace in exception.trace %}
-            {% set _is_vendor_trace = trace.file is not empty and ('/vendor/' in trace.file or '/var/cache/' in trace.file) %}
+            {% set _is_vendor_trace = trace.file is not empty and ('/vendor/' in trace.file or '/var/cache/' in trace.file or trace.file not in private_vendors) %}
```

Comment:
`trace.file not in private_vendors`

this requires the abs. file path to be configured? :thinking: shouldnt we test each private vendor separate?